packages/server/database/seeders/User.ts

```diff
@@ -23,19 +23,19 @@ export default class UserSeeder extends BaseSeeder {
             // Copy-pasted from packages/client/e2e/home.spec.ts
             {
                 uuid: '8d71dcb3-9638-4b7a-89ad-838e2310686c',
-                nickname: internet.userName(),
+                nickname: 'Francis',
             },
             {
                 uuid: '71bc3025-b765-4f84-928d-b4dca8871370',
-                nickname: internet.userName(),
+                nickname: 'Moris',
             },
             {
                 uuid: 'd125ecde-b0ee-4ab8-a488-c0e7a8dac7c5',
-                nickname: internet.userName(),
+                nickname: 'Leïla',
```

Comment:
I think setting constants names for theses users is necessary but i'm not sure about the given names
I think something more generic would be prefered
Such as available-user-A B C D etc ?

Reply:
To me, in E2E tests we are browsing our application as a real user would do.
And a real user would not be called _available-user-A_.
Cryptography has [Alice and Bob](https://en.wikipedia.org/wiki/Alice_and_Bob#Cast_of_characters), we could have our own characters! 😆

Reply:
Ok makes sense !
But could we be using an other than Leïla or maybe Leila as the diaeresis is quite a pain